core/src/forwarding_stage/packet_container.rs

```diff
@@ -1,11 +1,11 @@
-use {min_max_heap::MinMaxHeap, slab::Slab, solana_perf::packet::Packet};
+use {min_max_heap::MinMaxHeap, slab::Slab, solana_perf::packet::BytesPacket};
 
 /// Container for storing packets.
 /// Packet IDs are stored with priority in a priority queue and the actual
 /// `Packet` are stored in a map.
 pub struct PacketContainer {
     priority_queue: MinMaxHeap<PriorityIndex>,
-    packets: Slab<Packet>,
+    packets: Slab<BytesPacket>,
```

Comment:
Wondering if we can kind of just get rid of this container entirely now? and just use a `MinMaxHeap<(PriorityIndex, BytesPacket)>`. 

The original reason for a separate heap + map was because packets (with their inline data) had terrible perf in the heap since they were moving so often. That may not be the case now with BytesPacket since it's effectively just an `Arc`